### ISSUE TITLE
[serve] Make latency buckets configurable

### ISSUE THREAD

**Issue by zcin** [ray-project/ray#38223]
### Description

The latency buckets used for the latency histogram heavily affect how we calculate P50, P90 etc latency on Grafana. We should make this configurable so users who know roughly how long their requests will take can modify the buckets to match that.

**Comment by abrarsheikh** [comment]
## Prometheus Histogram Boundaries  

The Prometheus histogram boundaries are fetched from `DEFAULT_LATENCY_BUCKET_MS` in `constants.py`.  

### Current Usage of `Histogram` Metrics  
The `Histogram` metric is used in the following scenarios:  

1. **Model Load/Unload Times** – Tracks the time taken to load/unload multiplexed models.  
2. **Proxy-Level Request Latency** – Measures end-to-end request/response latency at the proxy, including both model inference time and infrastructure routing cost.  
3. **Replica-Level Request Latency** – Measures end-to-end request/response latency at the replica, primarily reflecting the isolated cost of model inference.  

### Decisions  

- Metrics **2 and 3** share similar characteristics in terms of histogram boundary definitions. A single configuration should control both.  
- Metric **1** is significantly different from **2 and 3**, so it requires a separate configuration for its histogram boundaries.  
- To maintain **backward compatibility**, `DEFAULT_LATENCY_BUCKET_MS` should be preserved for existing histogram-based metrics.  
- Introduce two new properties:  
  - `REQUEST_LATENCY_BUCKET_MS` for **metrics 2 and 3**.  
  - `MODEL_LOAD_LATENCY_BUCKET_MS` for **metric 1**.  

### Question  

Should the new properties be exposed via **environment variables** or as part of the **runtime configuration**, accessible through the public `deployment` decorator?

**Comment by zcin** [comment]
@abrarsheikh I agree that it should be configurable through environment variables instead of the deployment configuration. It's unlikely that users will often change this. Introducing two separate env vars `REQUEST_LATENCY_BUCKET_MS` and `MODEL_LOAD_LATENCY_BUCKET_MS` sgtm!
